### PR TITLE
project restructuring

### PR DESCRIPTION
**Pull request by dexter0175**
improving project architecture
SSR -> SPA
Updating dependencies
Configuring Webpack
all JS -> bundle.js
CSS -> bundle.css

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1,2 +1,2 @@
-web: npm run start
+web: npm run start
```
Comment (ops1ops) [inline Procfile]: `web: npm run start`

File: src/client/App.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import './app.css';
+import ReactImage from './react.png';
+
+export default class App extends Component {
```
Comment (ops1ops) [inline src/client/App.jsx]:
```
class App extends Component
...
export default App 
```

Comment (dexter0175) [reply src/client/App.jsx]: ?????????

File: src/server/routes/index.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { Router } from 'express';
+import { Op } from 'sequelize';
+import os from 'os';
+import db from '../db';
+
+const { Item } = db;
+const router = Router();
+
+router.get('/api/getUsername', (req, res) => res.send({ username: os.userInfo().username }));
+router.get('/api', async (req, res) => {
```
Comment (ops1ops) [inline src/server/routes/index.js]: `router.get('/api', async ({ query: { query } }, res) => {`

File: src/client/App.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        {username ? (
+          <h1>{`Hello ${username}`}</h1>
+        ) : (
+          <h1>Loading.. please wait!</h1>
+        )}
+        <img src={ReactImage} alt="react" />
+      </div>
+    );
+  }
+}
```
Comment (ops1ops) [inline src/client/App.jsx]:
```
}

export default App;
```

**Pull request merged**